packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -137,7 +139,7 @@ const AppContent = (): JSX.Element => {
       <RetryableTxnsIncluder />
       <TokenListSyncer />
       <BalanceUpdater />
-      <PWLoadedUpdater />
+      {!skipLoadingHistory && <PWLoadedUpdater />}
```

Comment:
I am against adding a query param specifically for Cypress testing purpose
generally we should encapsulate Cypress-specific code by detecting Cypress' existence
https://docs.cypress.io/faq/questions/using-cypress-faq#Is-there-any-way-to-detect-if-my-app-is-running-under-Cypress
we can do a conditional rendering here instead

Reply:
@spsjvc I find this approach to be much cleaner and resilient to end-user fiddling. I am going ahead with this for now, unless we find a strong reason not to do this.

Reply:
This is so much better! Thx @fionnachan